src/main/java/org/folio/de/entity/ExportConfigEntity.java

```diff
@@ -0,0 +1,59 @@
+package org.folio.de.entity;
+
+import java.util.List;
+import java.util.UUID;
+
+import org.folio.de.entity.base.AuditableEntity;
+import org.folio.des.domain.dto.ExportType;
+import org.hibernate.annotations.Type;
+
+import io.hypersistence.utils.hibernate.type.json.JsonBinaryType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.experimental.Accessors;
+
+@Entity
+@Table(name = "export_configs")
```

Comment:
the another table is named as job, to keep consistency we can name this one as export_config